staging/src/k8s.io/client-go/tools/clientcmd/client_config_test.go

```diff
@@ -132,9 +132,19 @@ func TestModifyContext(t *testing.T) {
 		"clean":   true,
 	}
 
+	tempPath, err := ioutil.TempFile("", "testclientcmd-")
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+	defer func() {
+		os.Remove(tempPath.Name())
```

Comment:
given this is a single call to be defered can we do?
```
defer os.Remove(tempPath.Name())
```

Reply:
Thanks, updated.